source/Energinet.DataHub.MeteringPoints.Domain/SeedWork/Internals/EnumerationReflection.cs

```diff
@@ -29,7 +29,7 @@ internal class EnumerationReflection
             EnumerationType[] types)
         {
             _types = types;
-            _nameLookup = types.ToDictionary(f => f.Name);
+            _nameLookup = types.ToDictionary(f => f.Name.ToLower());
```

Comment:
Use the invariant version of ToLower()

```suggestion
            _nameLookup = types.ToDictionary(f => f.Name.ToLowerInvariant());
```